packages/redux/src/contents/utils.ts

```diff
@@ -25,6 +25,40 @@ export enum ContentEnvironmentCode {
   Preview = 'preview',
 }
 
+/**
+ * Constant that represent the default object returned by content api without content
+ */
+const defaultContentReturn = {
+  number: 0,
+  totalItems: 0,
+  totalPages: 0,
+  entries: [],
+};
+
+/**
+ * Constant that represent the default section object returned by content api
```

Comment:
This is a function so I would rephrase this to something like: "Function that returns the default section which would be returned by content api containing content generated by Content Tool"

Reply:
Fixed. Thanks